src/greptimeai/utils/openai/parser.py

```diff
@@ -16,6 +17,18 @@ def _parse_choice(choice: Choice) -> Dict[str, Any]:
     return list([_parse_choice(choice) for choice in choices])
 
 
+def parse_completion_choices(
+    choices: List[CompletionChoice], verbose: bool = True
+) -> List[Dict[str, Any]]:
+    def _parse_choice(choice: CompletionChoice) -> Dict[str, Any]:
+        res = choice.model_dump()
+        if not verbose:
+            res.pop("message", None)
+        return res
+
+    return list([_parse_choice(choice) for choice in choices])
+
```

Comment:
Is it possible to merge `parse_completion_choices` with `parse_choices` into one function? function signature of `Union[...]` may help?